src/app/ui/font_popup.cpp

```diff
@@ -196,7 +197,8 @@ FontPopup::FontPopup()
 
   // Create one FontItem for each font
   for (auto& file : files) {
-    if (base::string_to_lower(base::get_file_extension(file)) == "ttf")
+    auto ext = base::string_to_lower(base::get_file_extension(file));
+    if (ext == "ttf" || ext == "otf")
```

Comment:
nitpicking: is `ext` the common short word for `extension`? I thought about `external` at a first glance, so I'm wondering if it makes more sense to just write the word in full

Reply:
Shortening `extension` into `ext` is pretty common throughout the codebase, although it does use the non-abbreviated form as well. :man_shrugging:

Reply:
..mm.